core/src/miner/backup.rs

```diff
@@ -69,11 +69,11 @@ pub fn recover_to_data(db: &KeyValueDB) -> RecoveredData {
 
         if b"item_" == key_prefix {
             let decoded_key = (key.as_ref()[5..]).into();
-            let decoded_item= rlp.as_val();
```

Comment:
These changes should be included in the previous commit.

Reply:
Moved that commit